src/value/mod.rs

```diff
@@ -25,13 +25,10 @@ pub enum Value {
     Null,
     /// Represents a boolean value.
     Bool(bool),
-    /// Integer CBOR numbers.
-    ///
-    /// The biggest value that can be represented is 2^64 - 1.
-    /// While the smallest value is -2^64.
-    /// Values outside this range can't be serialized
-    /// and will cause an error.
+    /// Signed integer CBOR numbers.
     Integer(i128),
+    /// Unsigned integer CBOR numbers.
+    Unsigned(u128),
```

Comment:
I don't like to have two overlapping integer variants (again) because a postive number can now be stored in two different ways.
Ist there something special about 128-bits that you need `u128` numbers that do not fit into `i128`?

Reply:
> Is there something special about 128-bits that you need u128 numbers that do not fit into i128?

The only issue is with the [canonical sort order](https://github.com/nhynes/cbor/blob/support-i128/src/value/mod.rs#L141). Before, `i128` could record the sign and magnitude of the largest supported number. Since `i128` is the largest data type, the sign isn't recorded independently of magnitude.